app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -41,62 +43,65 @@ const Notes = () => {
   }
 
   return (
-    <Container>
-      <Header
-        title="Notes"
-        actionBlock={
-          <Button
-            onClick={() => setShowNewNotePane(true)}
-            label="Add New Note"
-            icon="ri-add-line"
-          />
-        }
-        searchProps={{
-          value: searchTerm,
-          onChange: e => setSearchTerm(e.target.value),
-        }}
-      />
-      {notes.length ? (
-        <>
-          <SubHeader
-            rightActionBlock={
+    <div className="flex">
+      <MenuBar showMenu={isMenuBarOpen} />
+      <div
+        className={classnames({
+          "nc-list-wrapper": isMenuBarOpen,
+          "nc-list-wrapper-full": !isMenuBarOpen,
+        })}
+      >
+        <Container>
+          <Header
+            title="Notes"
+            actionBlock={
               <Button
-                label="Delete"
-                icon={Delete}
-                onClick={() => setShowDeleteAlert(true)}
-                disabled={!selectedNoteIds.length}
+                onClick={() => setIsNewNotePaneOpen(true)}
+                label="Add New Note"
+                icon="ri-add-line"
               />
             }
+            searchProps={{
+              value: searchTerm,
+              onChange: e => setSearchTerm(e.target.value),
+            }}
+            menuBarToggle={() => setIsMenuBarOpen(!isMenuBarOpen)}
```

Comment:
Please carefully read the following: https://www.bigbinary.com/react-best-practices/know-when-to-use-previous-state-for-state-update